### PR TITLE
Feature/user labels per document

### PR DESCRIPTION
**Pull request by RajanGuptaNZ**
#### Description
Added page to view user assigned labels for each document.
/project/:project/document/:document_id
ie: /project/New_Project/Document/5f6709e7e1991df825dafa96

#### Review Checklist
- [ ] The requirements on the description have been met
- [ ] The relevant documentation has been updated:
    - [ ] code comments for classes / methods / complex statements
    - [ ] package readme
    - [ ] documentation for high level approaches or product info
- [ ] Tests:
    - [ ] new tests written
    - [ ] existing affected tests updated
    - [ ] all green and passing
- [ ] Tested by reviewer

### CODE REVIEW

File: app/src/pages/DocumentPage.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+        })
+      } catch (e) {
+        console.log(e)
+      }
+    }, []);
+
+    useEffect(() => {
+      try {
+        projectServices.getDocument(project, document_id, firebase)
```
Comment (Cvos0162) [inline app/src/pages/DocumentPage.tsx:108]: when merging, use DocumentServices.
Let's match the structure to the backend API yeah?

File: app/src/pages/DocumentPage.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        .then(data => {
+            console.log(data)
```
Comment (Cvos0162) [inline app/src/pages/DocumentPage.tsx:110]: please remove all the console.log()

File: app/src/pages/DocumentPage.tsx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+                <IonLabel>{data.email}</IonLabel>
+                <IonLabel>{data.label}</IonLabel>
+              </IonItem>
+            ))}
+          </IonList>
+        </div>
+
+        <div className="container">
+            <IonButton className="ion-margin-top" type="submit" expand="block">
+                xxx
```
Comment (Cvos0162) [inline app/src/pages/DocumentPage.tsx:201]: What is this button for?

File: app/src/services/ProjectServices.ts [hunk 4]
```diff
@@ -89,6 +91,37 @@ async function getProjectUsers(project: string, firebase: any) {
         })
  }
 
+ function getDocument(project_name: any, document_id: any, firebase: any) {
```
Comment (Cvos0162) [inline app/src/services/ProjectServices.ts:94]: refer DocumentService

File: app/src/pages/DocumentPage.tsx [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+interface DocumentPageProps {
+    firebase: any
+  }
+
+interface Label {
+    _id: string;
+    name: string;
+}
+
+const users_and_labels_init: Users_and_Labels[] = [
```
Comment (Cvos0162) [inline app/src/pages/DocumentPage.tsx:51]: I dont think you need this
if you are concerned about type casting for useState
you can do useState<Users_and_Labels[]>([]) instread

**Review by Cvos0162 — CHANGES_REQUESTED** [review]
merge and clean up. Otherwise looks good to me 👍

**Comment by RajanGuptaNZ** [comment]
Made the requested changes

**Review by Cvos0162 — APPROVED** [review]
LGTM 👍

**Pull request merged**